views/templates/admin/tabs/help.tpl

```diff
@@ -0,0 +1,94 @@
+                    </p>
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">3</div>
+                    <div class="numberCircleText">
+                    <p class="numberCircleText">
+                        {l s='Allow your customer to consult and export their personal data collected by your shop on their customer account' d='Modules.Faviconnotificationbo.Admin'}
+                    </p>
+                    </div>
+                </div>
+                <div>
+                    <div class="numberCircle">4</div>
+                    <div class="numberCircleText">
+                    <p class="numberCircleText">
+                        {l s='View all your customers’ actions related to their personal data' d='Modules.Faviconnotificationbo.Admin'}
+                    </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="helpContentRight">
+            <div class="helpContentRight-sub">
+                <b>{l s='Need help ?' d='Modules.Faviconnotificationbo.Admin'}</b><br>
+                {l s='Find here the documentation of this module' d='Modules.Faviconnotificationbo.Admin'}
+                <a class="btn btn-primary" href="{$doc|escape:'htmlall':'UTF-8'}" target="_blank" style="margin-left:20px;" href="#">
+                    <i class="fa fa-book"></i> {l s='Documentation' d='Modules.Faviconnotificationbo.Admin'}</a>
+                </a>
+                <br><br>
+                <div class="tab-pane panel" id="faq">
+                    <div class="panel-heading"><i class="icon-question"></i> {l s='FAQ' d='Modules.Faviconnotificationbo.Admin'}</div>
+                    {foreach from=$apifaq item=categorie name='faq'}
+                        <span class="faq-h1">{$categorie->title|escape:'htmlall':'UTF-8'}</span>
+                        <ul>
+                            {foreach from=$categorie->blocks item=QandA}
+                                {if !empty($QandA->question)}
+                                    <li>
+                                        <span class="faq-h2"><i class="icon-info-circle"></i> {$QandA->question|escape:'htmlall':'UTF-8'}</span>
+                                        <p class="faq-text hide">
+                                            {$QandA->answer|escape:'htmlall':'UTF-8'|replace:"\n":"<br />"}
+                                        </p>
+                                    </li>
+                                {/if}
+                            {/foreach}
+                        </ul>
+                        {if !$smarty.foreach.faq.last}<hr/>{/if}
+                    {/foreach}
+                </div>
+                {l s='You couldn\'t find any answer to your question ?' d='Modules.Faviconnotificationbo.Admin'}
```

Comment:
Why this string? Is there any support planned for this module? I thought it was a native one...